dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -541,15 +541,16 @@ paths:
     post:
       tags:
       - "extendedGA4GH"
-      summary: "Update the index of tools"
-      description: "This endpoint updates the index for all published tools and workflows."
+      summary: "Update the workflows and tools indices"
+      description: "This endpoint updates the indices for all published tools and\
+        \ workflows."
       operationId: "toolsIndexGet"
       produces:
       - "text/plain"
       parameters: []
       responses:
         200:
-          description: "An array of Tools of the input organization."
+          description: "An array of Tools and an array of Workflows."
```

Comment:
Hmm...
- Is it returning two arrays?
- Is it returning one with multiple types?
- Perhaps it should return just the number of tools/workflows
- Maybe it should just not return any body, just a 200

Reply:
>     * Maybe it should just not return any body, just a 200

Not sure about the broader point, but if it doesn't return a body, it should technically return a 204.

Reply:
Hm technically it doesn't "return" anything, it just updates the index entries and the frontend loads those into arrays